### DISCUSSION TITLE
It takes a long time to export the gltf file of a large model.(20,000,000 Triangular，file size is about 1.5GB)

### DISCUSSION THREAD

**Discussion by shaoxiang0427** [assimp/assimp#5612] [Q&A] [answered]
It takes a long time(21minutes) to export the gltf file of a large model.(20,000,000 Triangular，file size is about 1.5GB)；The imported file is also a gltf file. Export it directly.

**Comment by kimkulling** [comment]
Hm, I guess we are currently not optimized to write bigger files. Have you used a debug build?

**Comment by kimkulling** [comment, marked as answer]
As a workaround disable the tangent calculation. This is really slow at this moment.
